### DISCUSSION TITLE
Web page will not scroll with deck.gl

### DISCUSSION THREAD

**Discussion by nilscb** [visgl/deck.gl#8101] [Q&A] [answered]
Imagine a web page with a deck.gl react component in it containing a couple of views/viewports but so that these views are smaller
than the full component leaving some space between them. If one scrolls using the wheel and mouse inside any of these views they will scroll their content as expected.

However if the mouse pointer is inside the deck.gl react component but outside the viewports in it, then one would expect the 
containing web page to scroll. That is;  the scroll events should be propagated from deck.gl to the underlying web page. This does not happen.

Would this be a defect in deck.gl?

Nils Christian

**Comment by felixpalmer** [comment, marked as answer]
Fixed in deck 8.9.31

**Reply by nilscb** [reply]
Good news! Thanks :-)
